Resources/Private/Templates/Formcycle.html

```
@@ -1,7 +1,50 @@
-<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" data-namespace-typo3-fluid="true">
+<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers"
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)